### DISCUSSION TITLE
next v0.12.14 release

### DISCUSSION THREAD

**Discussion by svenkreiss** [openpifpaf/openpifpaf#484] [General]
The next v0.12.14 release might be the last within v0.12. Afterwards, I would like to merge the `dev` branch and move to v0.13. Anything you want to get into v0.12.14, @dmizr @DuncanZauss @george-adaimi @bertoni9, like new pre-trained models or Guide additions or updates?

**Comment by dmizr** [comment]
@svenkreiss I have a few Swin pre-trained models with good performance on COCO val (68.4, 69.3 and 70.2 AP). However, the training had to be resumed, so there are 2 log files per model. Would they be worth uploading, and if so, how should I proceed?

**Reply by svenkreiss** [reply]
Yes, definitely interesting. We currently have no infrastructure to automatically handle log files. That they sit next to the checkpoint in a release is purely by convention. In your case, it would be nice to have both log files.

**Reply by dmizr** [reply]
Ok, great! I'm unfamilar with procedure to follow when releasing a pre-trained model, is there a document describing which steps to take (e.g. where to upload the files, how to name them, what files to modify, and how to compare their inference speed to the models described in the [intro](https://openpifpaf.github.io/intro.html))?

**Reply by svenkreiss** [reply]
The urls to pre-trained checkpoints are set in the plugin folder like this: https://github.com/openpifpaf/openpifpaf/blob/b2efc36598a8d15aafbcda14ec8a4312d88ee776/openpifpaf/plugins/coco/__init__.py#L13
The convention is to name them with `backbone-dataset` but for historical reason "cocokp" as dataset is omitted.
Once that is in, you can run the "benchmark" command as it is written under the table that you linked. You can provide other checkpoints once you define them as in the above link or even your own checkpoint files.

Also, I think this is a valid question, so if you have the time, please modify the Guide so that this is easier for the next person.

**Reply by dmizr** [reply]
Checkpoints can be found in PR #487
